File: Python/programmers/etc/level1/x만큼_간격이_있는_n개의_숫자.py
def solution(x, n):
    return [x*i for i in range(1, n+1)]


x, n = 2, 5
print(solution(x, n)) # [2,4,6,8,10]

x, n = 4, 3
print(solution(x, n)) # [4,8,12]

x, n = -4, 2
print(solution(x, n)) # [-4,-8]

File: Java/programmers/etc/level1/x만큼간격이있는n개의숫자.java
package CodeTest.Java.programmers.etc.level1;

public class x만큼간격이있는n개의숫자 {
    public long[] solution(int x, int n) {
        long[] answer = new long[n];
        answer[0] = x;

        for (int i=1; i<n; i++) {
            answer[i] = answer[i-1] + x;
        }

        return answer;
    }
}
